ocaml/stdlib/camlinternalOO.ml

```diff
@@ -291,7 +295,9 @@ let new_methods_variables table meths vals =
   res
 
 let get_variable table name =
-  try Vars.find name table.vars with Not_found -> assert false
+  match Vars.find_opt name table.vars with
+  | Some x -> x
+  | None -> assert false
```

Comment:
This part of the patch seems less useful, since no exceptions are ever raised. Although setting up the exception handler might already be expensive enough to justify the switch to options.

Reply:
Good point. I don't believe that setting up the handler is the expensive part.

Reply:
(This is now reverted.)